JDP Remediation Tool 10.0 Release

VirendraKumarSharePoint opened the pull request:
1. Updated below commands: 
- Generate Site Column & Content Type Usage Report 
- Generate Non-Default Master Page Usage Report 
- Generate Site Collection Report (PPE-Only)
2. Fixed Login issue 
3. Implemented Success/Failure CSV in all the commands

virendrak committed:
JDP Remediation Tool 10.0 Release

1. Updated below commands:
o Generate Site Column & Content Type Usage Report
o Generate Non-Default Master Page Usage Report
o Generate Site Collection Report (PPE-Only)
2. Fixed Login issue
3. Implemented Success/Failure CSV in all the commands

virendrak committed:
JDP Remediation Tool 10.0 Release

1. Updated below commands:
o Generate Site Column & Content Type Usage Report
o Generate Non-Default Master Page Usage Report
o Generate Site Collection Report (PPE-Only)
2. Fixed Login issue
3. Implemented Success/Failure CSV in all the commands

virendrak committed:
JDP Remediation Tool 10.0 Release

1. Updated below commands:
o Generate Site Column & Content Type Usage Report
o Generate Non-Default Master Page Usage Report
o Generate Site Collection Report (PPE-Only)
2. Fixed Login issue
3. Implemented Success/Failure CSV in all the commands

msftclas commented:
Hi __@VirendraKumarSharePoint__, I'm your friendly neighborhood Microsoft Pull Request Bot (You can call me MSBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.microsoft.com.
    </span>

TTYL, MSBOT;

virendrak committed:
JDP Remediation Tool 10.0 Release

1. Updated below commands:
o Generate Site Column & Content Type Usage Report
o Generate Non-Default Master Page Usage Report
o Generate Site Collection Report (PPE-Only)
2. Fixed Login issue. Giving 3 attempts to user if he provides the wrong
login id/password
3. Implemented Success/Failure CSV in all the commands
4. Optimized the List and Ste Template Crack Open logic, so that we
won’t get the Long path error while saving and WSP/STP and reading the
content for these files.

jansenbe commented:
Thx Virendra.
